apps/src/code-studio/components/shareDialogRedux.js

```diff
@@ -35,8 +35,10 @@ export default function reducer(state = initialState, action) {
       };
     case UNPUBLISH_SUCCESS:
       return {
-        ...initialState,
+        ...state,
```

Comment:
it looks like you're setting `isOpen` and `isUnpublishPending` to the defaults in `initialState`, so why spread `state` instead of keeping `initialState`? i'm guessing there's additional state that needs to be maintained here, but i'm wondering what it is

Reply:
Yes, there's additional state, the projects list, that I need to keep around so that I can update the recently unpublished project's publishedAt field.